src/Illuminate/Session/Console/stubs/database.stub

```diff
@@ -14,7 +14,7 @@ class CreateSessionsTable extends Migration
     public function up()
     {
         Schema::create('sessions', function (Blueprint $table) {
-            $table->string('id')->primary();
+            $table->id()->primary();
```

Comment:
Session ID is a string and can contain letters - `U9YE1dk4GOfDk6Tdh9Rqs67SHETvzG5Cxh0magUe`.

- https://github.com/laravel/framework/blob/8.x/src/Illuminate/Session/Store.php#L19
- https://www.php.net/manual/en/class.sessionhandlerinterface.php

Reply:
I understood I thought of it as an "id".